### DISCUSSION TITLE
💡 Stories scraping as guest

### DISCUSSION THREAD

**Discussion by ithimea** [diezo/Ensta#34] [Ideas]
Hello, nice Project!

As a possibility of improvement/Idea, could story scraping be implemented? Since there are many websites that do this, however I don't know if this is through the use of many accounts or directly scraping IG.
Because, in those webs you can do many requests without problem, so they have multiple accounts using proxies, or a method to do direct scraping without being blocked.

But anyway, thanks and nice repo.

**Comment by diezo** [comment]
Thanks for the suggestion and those nice words. I'll soon implement story scraping into the host class of ensta.

If you try to view someone's story in incognito mode, you'll realise that you can't do that without being logged in.

So you'll need to use a dedicated service that uses different ghost accounts to scrape data without being restricted.

**Comment by diezo** [comment]
If I launch such a service in the near future, I'll let you know.

Btw it would be great if you give this repository a star so that more developers can know about ensta.

Thankyou

**Comment by ithimea** [comment]
Yeah, I tried something similar, when u mentioned viewing a story in incognito, so u need the account to get the data displayed. So I finally used other methods for the scrape of the data.
However, ur project looks good, thanks for answering my questions! (and I had forgotten the ⭐ hahaha)
